PWG/EMCAL/EMCALtrigger/AliEmcalTriggerSimQATask.cxx

```diff
@@ -209,8 +259,16 @@ void AliEmcalTriggerSimQATask::DoClusterLoop() {
         continue;
       }
 
+      // Get Cluster vector
+      TLorentzVector vCluster;
+      clusters->GetMomentum(vCluster,cluster);
+      Double_t fPhi = vCluster.Phi();
+      if (fPhi < 0) fPhi+=2*TMath::Pi();
```

Comment:
FYI: If you use AliTLorentzVector, you can call the function AliTLorentzVector::Phi_0_2pi()